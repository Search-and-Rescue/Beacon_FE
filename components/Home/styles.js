import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  home: {
    marginTop: 80,
    backgroundColor: "grey"
  },
  homeBtns: {
    borderWidth: 3,
    borderColor: 'red',
    height: 250,
    marginHorizontal: 30,
    marginVertical: 20,
    width: 300
  },
  addTripBtn: {
    fontSize: 38,
    marginTop: 20
  },
  editTripBtn: {
    fontSize: 38,
    marginTop: 20
  }
});

export default styles;