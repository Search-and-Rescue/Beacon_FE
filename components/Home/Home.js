import React from 'react';
import { View, Text, TouchableOpacity } from 'react-native';
import styles from './styles';

import MenuButton from "../../navigation/MenuButton/MenuButton";

const Home = (props) => {
  return (
    <View>
      <MenuButton navigation={props.navigation} />
      <View style={styles.home}>
        <View style={styles.homeBtns}>
          <TouchableOpacity title="Add Trip">
            <Text style={styles.addTripBtn}>Add Trip</Text>
          </TouchableOpacity>
        </View>
        <View style={styles.homeBtns}>
          <TouchableOpacity title="History/Edit Trip">
            <Text style={styles.editTripBtn}>Edit Trip History</Text>
          </TouchableOpacity>
        </View>
      </View>
    </View>
  );
}

export default Home;