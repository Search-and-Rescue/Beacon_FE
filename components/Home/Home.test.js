import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme';

describe('Home', () => {
  it('should create a snapshot of the component with the correct data passed through', () => {
    expect(shallow(<Home />)).toMatchSnapshot();
  });
});