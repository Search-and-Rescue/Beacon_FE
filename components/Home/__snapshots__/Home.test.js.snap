// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should create a snapshot of the component with the correct data passed through 1`] = `
<View
  style={Object {}}
>
  <View
    style={Object {}}
  >
    <Button
      title="Add Trip"
    />
  </View>
  <View
    style={Object {}}
  >
    <Button
      title="History/Edit Trip"
    />
  </View>
</View>
`;
